proto/p4/config/p4info.proto

```diff
@@ -78,17 +78,22 @@ message Table {
   repeated int32 action_ids = 3;
   // 0 (default value) means that the table does not have a const default action
   int32 const_default_action_id = 4;
+  // a table may have a const default action, whose action parameter values can
+  // be changed at runtime. However, in most cases a table's action entry is set
+  // in the P4 program and cannot be changed, which is what this boolean flag
+  // indicates.
+  bool has_const_default_action_entry = 5;
```

Comment:
This comment and field name is a bit unclear to me even though I know why you're adding this. =) I feel like our terminology is inconsistent (action params / action data / action entry), which may be the problem. Maybe rename to something more verbose but clearer like `const_default_action_has_mutable_params` and update the comment to state more clearly that the issue is whether it's possible to change the parameters to the const default action.

This is so awkward that I wonder if it's even the right approach. Maybe when the action parameters cannot be changed, we should just wrap the original action in a synthesized an action that has no parameters.